src/main/java/games/strategy/engine/random/PropertiesDiceRoller.java

```diff
@@ -247,3 +256,28 @@ public String getHelpText() {
     return getInfoText();
   }
 }
+
+
+class AdvancedRedirectStrategy extends LaxRedirectStrategy {
```

Comment:
So does that mean that the extension classes at the bottom of the delegates violate the checkstyle?